src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -344,6 +344,10 @@ protected override Size ArrangeOverride(Size finalSize)
 					UpdateLayout(ref _measureLayout, arrangeSize, exactWidth: true);
 				}
 
+				// Paint depends on LayoutSlot, which isn't yet set when in MeasureOverride.
```

Comment:
I think we shouldn't depend on LayoutSlot. We want to somehow get the measurement of the text itself, not the textblock as a whole, ie, the TextBlock may be taking the whole width, while the text doesn't entirely fill it.


@carldebilly Since you recently worked on some size-related stuff, do you know of a good fix here?

I tried things like `_paint.GetTextBounds(_textFormatted, 0, _textFormatted.Length(), rect);` but the rect turned out having negative values and things went wrong.